### ISSUE TITLE
notes populating incorrectly?

### ISSUE THREAD

**Issue by dwhieb** [dwhieb/Nuuchahnulth#1]
This utterance:

```txt
# 249
waamitii           tušaakitʔi.
wa·-mit-(y)iː      tušaːk-it-ʔi·
say-PAST-INDF.3    dead?-PAST-DEF
he.might.have.said corpse
He might have said, “you SOB.”
\n DWH: SOB = son of a bitch
```

Was parsed this way:

```json
    {
      "metadata": "249",
      "transcription": "waamitii tušaakitʔi.",
      "translation": "he.might.have.said corpse",
      "words": [
        {
          "transcription": "",
          "analysis": "wa·-mit-(y)iː",
          "gloss": "say-PAST-INDF.3",
          "morphemes": [
            {
              "transcription": "wa·",
              "gloss": "say"
            },
            {
              "transcription": "mit",
              "gloss": "PAST"
            },
            {
              "transcription": "(y)iː",
              "gloss": "INDF.3"
            }
          ],
          "tags": {
            "morphPOS": "PRED-REF"
          }
        },
        {
          "transcription": "",
          "analysis": "tušaːk-it-ʔi·",
          "gloss": "dead?-PAST-DEF",
          "morphemes": [
            {
              "transcription": "tušaːk",
              "gloss": "dead?"
            },
            {
              "transcription": "it",
              "gloss": "PAST"
            },
            {
              "transcription": "ʔi·",
              "gloss": "DEF"
            }
          ],
          "tags": {
            "morphPOS": "REF"
          }
        }
      ],
      "notes": [
        {
          "language": "en",
          "text": "He might have said, “you SOB.”"
        },
        {
          "language": "en",
          "source": "DWH",
          "text": "SOB = son of a bitch"
        }
      ],
      "n-4": "He might have said, “you SOB.”",
      "n-5": "DWH: SOB = son of a bitch"
    }
```

**Comment by dwhieb** [comment]
This needs to be fixed in the scription2dlx library.

**Comment by dwhieb** [comment]
This should now be fixed in scription2dlx@v0.11.0.
